src/lib/raft/raft.c

```diff
@@ -1089,7 +1089,7 @@ raft_checkpoint_remote(const struct raft *raft, struct raft_msg *req)
 	req->vote = raft->vote;
 	req->state = raft->state;
 	req->leader_id = raft->leader;
-	req->is_leader_seen = raft_is_leader_seen(raft);
+	req->is_leader_seen = raft->is_enabled && raft_is_leader_seen(raft);
```

Comment:
Please leave a short comment here why we don't want the disabled nodes to report `is_leader_seen`.

Reply:
Done, let me know what you think of this one